pkg/client/client.go

```diff
@@ -325,7 +328,26 @@ func New(c *restclient.Config) (*Client, error) {
 		return nil, err
 	}
 
-	return &Client{client}, nil
+	return &Client{RESTClient: client, config: &config}, nil
+}
+
+// ForGroupVersion returns a copy of the client but without any openshift legacy
```

Comment:
How is this client different than a client created by kube?  Couldn't we start collapsing onto theirs with moderate adjustments for GroupVersion?